Can't contribute as unable to install node_modules

### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

I'm trying to fork this repo so I can contribute via pull requests. I've tried to follow the Contribution guide on the SvelteUI website, but I'm unable to run `npm install` at the root of the monorepo due to dependency errors shown below;

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @svelteuidev/composables@0.7.5
npm ERR! Found: vite@3.0.0
npm ERR! node_modules/vite
npm ERR!   dev vite@"3.0.0" from @svelteuidev/composables@0.7.5
npm ERR!   packages/svelteui-composables
npm ERR!     @svelteuidev/composables@0.7.5
npm ERR!     node_modules/@svelteuidev/composables
npm ERR!       workspace packages/svelteui-composables from the root project
npm ERR!       3 more (@svelteuidev/prism, @svelteuidev/core, @svelteuidev/dates)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^3.1.0" from @sveltejs/kit@1.0.0-next.517
npm ERR! node_modules/@sveltejs/kit
npm ERR!   dev @sveltejs/kit@"next" from @svelteuidev/composables@0.7.5
npm ERR!   packages/svelteui-composables
npm ERR!     @svelteuidev/composables@0.7.5
npm ERR!     node_modules/@svelteuidev/composables
npm ERR!       workspace packages/svelteui-composables from the root project
npm ERR!       3 more (@svelteuidev/prism, @svelteuidev/core, @svelteuidev/dates)
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/kyleshepherd/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kyleshepherd/.npm/_logs/2022-11-04T15_15_24_253Z-debug-0.log
```

### In which browser(s) did the problem occur?

_No response_

### Steps To Reproduce

1. Clone repo
2. Run `npm install` at root of monorepo
3. See deps error

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

### Relevant Assets

_No response_

if I attempt to run `npm install --legacy-peer-deps` I get the below error:
```
npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated kleur@2.0.2: Please upgrade to kleur@3 or migrate to 'ansi-colors' if you prefer the old syntax. Visit <https://github.com/lukeed/kleur/releases/tag/v3.0.0\> for migration path(s).
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm WARN deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
npm ERR! code 127
npm ERR! path /Users/kyleshepherd/repos/svelteui/apps/docs
npm ERR! command failed
npm ERR! command sh /var/folders/f2/2q5w93yx7377vm12v23mbpl80000gn/T/prepare-86f97a24.sh
npm ERR! /var/folders/f2/2q5w93yx7377vm12v23mbpl80000gn/T/prepare-86f97a24.sh: line 1: deno: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kyleshepherd/.npm/_logs/2022-11-04T16_24_15_857Z-debug-0.log
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -66,6 +66,9 @@ Examples:
 1. cd into the package you want to develop on
 2. run `yarn dev`
 
+**If developing in docs**
+
+- Install [`deno`]https://deno.land/#installation
 - To start docs - `cd docs && yarn install && yarn dev`
 
 ## Yarn scripts

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -68,7 +68,7 @@ Examples:
 
 **If developing in docs**
 
-- Install [`deno`]https://deno.land/#installation
+- Install [`deno`](https://deno.land/#installation)
 - To start docs - `cd docs && yarn install && yarn dev`
 
 ## Yarn scripts

--- a/apps/docs/package.json
+++ b/apps/docs/package.json
@@ -16,12 +16,12 @@
   "scripts": {
     "build": "rollup -c ../../node_modules/@svelte-docs/core/rollup.config.js",
     "autobuild": "rollup -c ../../node_modules/@svelte-docs/core/rollup.config.js -w",
-    "prepare": "deno run --unstable --allow-read --allow-write ./prepare.ts",
-    "dev": "yarn prepare & yarn start:dev & yarn autobuild",
+    "prepare:docs": "deno run --unstable --allow-read --allow-write ./prepare.ts",
+    "dev": "yarn prepare:docs & yarn start:dev & yarn autobuild",
     "start": "node ../../node_modules/@svelte-docs/server",
     "start:dev": "PORT=3002 node ../../node_modules/@svelte-docs/server --dev --single",
     "start:pagewatch": "node ../../node_modules/@svelte-docs/core/watcher",
-    "deploy": "yarn prepare & yarn build && deno run -A --unstable ../../scripts/docs/build.ts",
+    "deploy": "yarn prepare:docs & yarn build && deno run -A --unstable ../../scripts/docs/build.ts",
     "deploy:nobuild": "node ../../node_modules/@svelte-docs/publisher"
   }
 }

--- a/packages/svelteui-composables/package.json
+++ b/packages/svelteui-composables/package.json
@@ -76,6 +76,6 @@
 		"svelte-preprocess": "4.10.6",
 		"tslib": "2.3.1",
 		"typescript": "4.7.4",
-		"vite": "3.0.0"
+		"vite": "3.2.2"
 	}
 }

--- a/packages/svelteui-core/package.json
+++ b/packages/svelteui-core/package.json
@@ -83,7 +83,7 @@
 		"svelte2tsx": "0.5.7",
 		"tslib": "2.3.1",
 		"typescript": "4.7.4",
-		"vite": "3.0.0"
+		"vite": "3.2.2"
 	},
 	"peerDependencies": {
 		"@svelteuidev/composables": "0.7.5",

--- a/packages/svelteui-dates/package.json
+++ b/packages/svelteui-dates/package.json
@@ -80,7 +80,7 @@
 		"svelte2tsx": "0.5.7",
 		"tslib": "2.3.1",
 		"typescript": "4.7.4",
-		"vite": "3.0.0"
+		"vite": "3.2.2"
 	},
 	"peerDependencies": {
 		"@svelteuidev/composables": "0.7.5",

--- a/packages/svelteui-demos/package.json
+++ b/packages/svelteui-demos/package.json
@@ -81,6 +81,6 @@
 		"svelte2tsx": "0.5.7",
 		"tslib": "2.3.1",
 		"typescript": "4.7.4",
-		"vite": "3.0.0"
+		"vite": "3.2.2"
 	}
 }

--- a/packages/svelteui-motion/package.json
+++ b/packages/svelteui-motion/package.json
@@ -70,7 +70,7 @@
 		"svelte-preprocess": "4.10.6",
 		"tslib": "2.3.1",
 		"typescript": "4.7.4",
-		"vite": "3.0.0"
+		"vite": "3.2.2"
 	},
 	"peerDependencies": {
 		"svelte": ">=3.47.0"

--- a/packages/svelteui-preprocessors/package.json
+++ b/packages/svelteui-preprocessors/package.json
@@ -79,7 +79,7 @@
 		"svelte2tsx": "0.5.7",
 		"tslib": "2.3.1",
 		"typescript": "4.7.4",
-		"vite": "3.0.0",
+		"vite": "3.2.2",
 		"vitest": "0.13.0"
 	},
 	"peerDependencies": {

--- a/packages/svelteui-prism/package.json
+++ b/packages/svelteui-prism/package.json
@@ -81,7 +81,7 @@
 		"svelte2tsx": "0.5.7",
 		"tslib": "2.3.1",
 		"typescript": "4.7.4",
-		"vite": "3.0.0"
+		"vite": "3.2.2"
 	},
 	"peerDependencies": {
 		"@svelteuidev/composables": "0.7.5",

--- a/packages/svelteui-tests/package.json
+++ b/packages/svelteui-tests/package.json
@@ -65,6 +65,6 @@
 		"sass": "1.50.0",
 		"svelte-preprocess": "4.10.6",
 		"typescript": "4.7.4",
-		"vite": "3.0.0"
+		"vite": "3.2.2"
 	}
 }

